src/test/java/org/wiremock/integrations/testcontainers/WireMockContainerJUnit5Test.java

```diff
@@ -1,55 +1,83 @@
 import org.testcontainers.junit.jupiter.Testcontainers;
 import org.wiremock.integrations.testcontainers.testsupport.http.TestHttpClient;
 
+import java.net.http.HttpResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
 @Testcontainers
 public class WireMockContainerJUnit5Test {
 
-  @Container
-  public WireMockContainer wiremockServer = new WireMockContainer("2.35.0")
-      .withMapping("hello", WireMockContainerTest.class, "hello-world.json")
-      .withMapping("hello-resource", WireMockContainerTest.class, "hello-world-resource.json")
-      .withFileFromResource("hello-world-resource-response.xml", WireMockContainerTest.class,
-          "hello-world-resource-response.xml");
-
-
-  @ParameterizedTest
-  @ValueSource(strings = {
-          "hello",
-          "/hello"
-  })
-  public void helloWorld(String path) throws Exception {
-    // given
-    String url = wiremockServer.getUrl(path);
-
-    // when
-    HttpResponse<String> response = TestHttpClient.newInstance().get(url);
-
-    // then
-    assertThat(response.body())
-        .as("Wrong response body")
-        .contains("Hello, world!");
-  }
-
-  @Test
-  public void helloWorldFromFile() throws Exception {
-    // given
-    String url = wiremockServer.getUrl("/hello-from-file");
-
-    // when
-    HttpResponse<String> response = TestHttpClient.newInstance().get(url);
-
-    // then
-    assertThat(response.body())
-        .as("Wrong response body")
-        .contains("Hello, world!");
-  }
+    @Container
+    public WireMockContainer wiremockServer = new WireMockContainer("2.35.0")
+            .withMapping("hello", WireMockContainerTest.class, "hello-world.json")
+            .withMapping("hello-resource", WireMockContainerTest.class, "hello-world-resource.json")
+            .withFileFromResource("hello-world-resource-response.xml", WireMockContainerTest.class,
+                    "hello-world-resource-response.xml");
+
+
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "hello",
+            "/hello"
+    })
+    public void helloWorld(String path) throws Exception {
+        // given
+        String url = wiremockServer.getUrl(path);
+
+        // when
+        HttpResponse<String> response = TestHttpClient.newInstance().get(url);
+
+        // then
+        assertThat(response.body())
+                .as("Wrong response body")
+                .contains("Hello, world!");
+    }
+
+    @Test
+    public void helloWorldFromFile() throws Exception {
+        // given
+        String url = wiremockServer.getUrl("/hello-from-file");
+
+        // when
+        HttpResponse<String> response = TestHttpClient.newInstance().get(url);
+
+        // then
+        assertThat(response.body())
+                .as("Wrong response body")
+                .contains("Hello, world!");
+    }
+
+    @Test
+    public void defaultBannerDisabled() {
```

Comment:
If you do not rely `@Container`, better to create a separate test class like `WireMockContainerUnitTest` that does not use annotations or spin up the server instance